modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspaceExtension.java

```diff
@@ -111,20 +142,35 @@ public String doCall() {
 			}
 
 		};
+
 		_dockerImageLiferay = _getProperty(
-			settings, "docker.image.liferay", _DOCKER_IMAGE_LIFERAY);
+			settings, "docker.image.liferay", _getDefaultDockerImage());
+
 		_environment = _getProperty(
 			settings, "environment",
 			BundleSupportConstants.DEFAULT_ENVIRONMENT);
+
 		_homeDir = _getProperty(
 			settings, "home.dir",
 			BundleSupportConstants.DEFAULT_LIFERAY_HOME_DIR_NAME);
+
+		_liferayProductVersion = _getProperty(
+			settings, "product.version", _getDefaultLiferayProductVersion());
+
+		_releaseDate = _getProperty(
```

Comment:
We don't need to store anything besides the `_product` because everything else can be obtained from the `_product`.  So we can remove both `_releaseDate` and `_liferayProductVersion`